### DISCUSSION TITLE
飞布内测招募

### DISCUSSION THREAD

**Discussion by AnsonCode** [fireboomio/product-manual#1] [Announcements]
飞布将于2023年2月10日开始内测，邀请感兴趣的同学一起参与。

添加微信，联系飞布产品负责人，参与内测或了解产品动态。

![image](https://user-images.githubusercontent.com/20960032/215998385-c404d75a-0d7d-49f8-9334-b00dceb4692b.png)

**Comment by electricface** [comment]
可以用 chatgpt 配合写代码吗？

**Reply by huyongchao98** [reply]
敬请期待！

**Reply by AnsonCode** [reply]
> 可以用 chatgpt 配合写代码吗？

欢迎 加入内测群，近期会有ai能力集成的更新！

**Reply by AnsonCode** [reply]
> 可以用 chatgpt 配合写代码吗？

目前已经支持了，欢迎体验：https://www.bilibili.com/video/BV1gb411o7NC/?spm_id_from=333.999.0.0

**Comment by 729902288** [comment]
您好，请问支持mongodb数据库了吗？

**Reply by AnsonCode** [reply]
你好，mongodb在排期中，可以加我微信，了解最新动态。

**Reply by erguotou520** [reply]
你好，mongodb已经支持了，迁移功能暂不支持

**Comment by SaraiQX** [comment]
请问有mac版吗？试了下curl命令 貌似不支持mac呢 😭 感谢

**Reply by AnsonCode** [reply]
支持的，用官网的安装命令试一试，或者用docker镜像试一下。

**Comment by foven** [comment]
连接数据源时，后台日志提示You defined no datasource. You must define exactly one datasource.

**Reply by AnsonCode** [reply]
在数据源里面，添加一个数据源即可。具体 可以进群询问。

**Reply by foven** [reply]
请问怎么加群，那个二维码失效了

**Reply by erguotou520** [reply]
@foven 因群人数超出限制，可以先加微信 luxuncode，然后拉你进群

**Comment by epver** [comment]
建议上个飞书，飞书有5000人的群

**Comment by conglinyizhi** [comment]
Application error: a client-side exception has occurred (see the browser console for more information).
访问 URL （ https://doc.fireboom.io/geng-xin-ri-zhi/v2.0-geng-xin-shuo-ming ）报告异常
